orchestrator/nodes/dockerComposeRemoteNode/index.js

```diff
@@ -48,13 +48,20 @@ class DataHandler extends RemoteNode {
       }
 
       this.client.networks().list().then((results) => {
-        for (let result of results) {
-          let name = result.Name.match(/.+?_flowbroker/);
-          if (name) {
-            this.targetNetwork = name;
-            return resolve(name);
+        let network = process.env.FLOWBROKER_NETWORK;
```

Comment:
I would prefer that the network information become a class attribute and the assignment occurs on class constructor.
The reason is class design and code maintainability.